CanvasColorSpaceProposal.md

```diff
@@ -167,16 +167,16 @@ interface ImageData {
 
 * When using the constructor that takes an ImageDataArray parameter, the "storageType" setting is ignored.
 * createImageData() and getImageData() produce an ImageData object with the same color space as the source canvas, using an ImageDataArray of a type that is appropriate for the pixelFormat of the source canvas (smallest possible numeric size that guarantees no loss of precision).
-* If the canvas color space is not "srgb" or the pixel format is not "8-8-8-8", the data returned by getImageData() is in linear gamma color space.
+* If the canvas color space is not "srgb" or the pixel format is not "8-8-8-8", the data returned by getImageData() is in linear color space.
 * putImageData() performs a color space conversion to the color space of the destination canvas.
-* Data returned by getImageData() or passed to putImageData() are assumed to be in linear gamma color space.
-* If the image data color space is not "srgb" or the storage format is not "uint8", the data passed to putImageData() is assumed to be in linear gamma color space.
+* Data returned by getImageData() or passed to putImageData() are assumed to be in linear color space.
```

Comment:
Is this line actually true? I suspect the correct thing to say is non-linear (encoded) space, because the next line appears to then be describing the exception to the rule established here. (And for 8888 sRGB data, I would strongly believe that put/getImageData operate on encoded values).